packages/core/src/components/button/_button.scss

```diff
@@ -260,6 +261,17 @@ a.pt-button {
   @extend .pt-button:disabled;
 }
 
+.pt-button-loading-spinner {
```

Comment:
I feel like it'd be neat to select this spinner via composition instead of custom class: `.pt-button .pt-spinner`